docs/meshstack.aws.md

```diff
@@ -6,3 +6,84 @@ title: Amazon Web Services
+
+Meshstack automatically configures AWS IAM in all managed accounts to integrate SSO with [Meshstack Identity Federation](./meshstack.identity-federation.md).
+
+## Prerequisites
+
+### AWS Root Account
+
+Meshstack uses [AWS Organizations](https://aws.amazon.com/organizations/) to provision and manage AWS Accounts for [Meshprojects](./meshcloud.project.md). To use AWS with a Meshstack deployment, operators will need an AWS "root" account acting as the parent of all accounts managed by Meshstack.
+
+### Identifier Configuration
+
+Meshstack operators that want to use AWS must configure their deployment to restrict identifier lengths to meet AWS requirements. The maximum allowed lengths are:
+
+```yaml
+customer_identifier_length: 16
+project_identifier_length: 30
+```
+
+### Meshstack IAM User
+
+The Meshstack AWS Connector uses a dedicated set of IAM credentials to work with AWS APIs on behalf of meshstack. To create these credentials, create a user in IAM with these specifications:
+
+- User name: meshfed-service
+- AWS access type: Programmatic access - with an access key
+
+Attach the following inline policy using this json
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "OrganizationsFullAccess",
+            "Effect": "Allow",
+            "Action": "organizations:*",
+            "Resource": "*"
+        },
+        {
+            "Sid": "StsAccessMemberAccount",
+            "Effect": "Allow",
+            "Action": "sts:AssumeRole",
+            "Resource": "arn:aws:iam::*:role/OrganizationAccountAccessRole"
+        },
+        {
+            "Sid": "IamFullAccess",
+            "Effect": "Allow",
+            "Action": "iam:*",
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+Operators need to securely inject the generated credentials into the configuration of the AWS Connector.
+
+### Project-Account Email Addresses
+
+AWS requires a unique email address for each account. Operators must thus configure a wildcard email address pattern with a placeholder `%s`. The pattern must not exceed a total length of `20` characters (including the placeholder). For example, this pattern
+
+```yaml
+accountEmailTemplate: aws+%s@meshcloud.io
+```
+
+allows generation of account names:
+
+- aws+customer.projectA@meshcloud.io
+- aws+customer.projectB@meshcloud.io
+
+### IAM Roles and Service Control Policies
+
+When a Meshstack user accesses an AWS project, they are assigned an AWS IAM role based on their project role configured on the Meshproject. Operators can configure these roles and their permissions by providing an [AWS Cloud Formation](https://aws.amazon.com/cloudformation/) template.
+
+When configuring these roles, operators must take care to correctly guard against privilege escalation and maintain project sandboxing. Operators should also consider leveraging [Service Control Policies](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scp.html) and [...].
```

Comment:
@JohannesRudolph The last part of the sentence is missing ("and ..."), please complete

- [ ] f